examples/quickstarts/helidon-quickstart-mp/build.gradle

```diff
@@ -16,7 +16,7 @@
 
 plugins {
     id 'java'
-    //id 'org.kordamp.gradle.jandex' version '0.5.0'
+    id 'org.kordamp.gradle.jandex' version '0.5.0'
```

Comment:
Version `0.6.0` is out, which makes sure the jandex file is placed automatically under `META-INF` just as its Maven counterpart does.

Reply:
I've upgraded to 0.6.0